app/assets/javascripts/comments.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
@samehta 
Blank asset files like `comments.coffee` can be safely deleted.

If you want to avoid manually deleting these blank generated files, you can add some configuration to `config/application.rb` that'll turn off the generators you don't want. This will stop these blank asset files from being created.

Should you want to do that, you can add the following `config.generators do ...` block towards the end of this section of `config/application.rb`:

```ruby
module YourAppNameIsAlreadyHere
  class Application < Rails::Application
    ...
    # These are the lines you add:
    config.generators do |g|
      g.assets = false # Prevents generation of blank asset files
    end

```